### ISSUE TITLE
Validation accuracy is higher than training accuracy

### ISSUE THREAD

**Issue by Sebastian-X** [gabrielhuang/reptile-pytorch#6]
Hi, thanks for your implementation!
When I run you code, I found that accuracy of "Meta-Val" dataset starts at a very high level around 0.7
, whereas accuracy of "Meta-Train" set is only 0.5. As the training goes on, this gap reduces but val accuracy is still slightly higher than train accuracy. Is this a normal phenomenon?
![IWVPR)WEGAU8N%ISMII`S76](https://user-images.githubusercontent.com/58028948/73331544-1c48fc80-429f-11ea-9a65-ec7fe90b66fe.png)
